app/components/icons/article-icon.tsx

```diff
@@ -0,0 +1,46 @@
+export const ArticleIcon = ({
```

Comment:
it should be removed, right?

Reply:
yeah, I've removed on this commit 6498cb1c2c1e08ee2dad5d6f1b0b36904e0b7857, maybe there something wrong with my merge. I'll fix it